cmake/pcl_find_cuda.cmake

```diff
@@ -31,25 +22,36 @@ if(CUDA_FOUND)
   # https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#deprecated-features
   
   if(NOT ${CUDA_VERSION_STRING} VERSION_LESS "11.0")
-    set(__cuda_arch_bin "3.5 3.7 5.0 5.2 5.3 6.0 6.1 6.2 7.0 7.2 7.5 8.0")
+    set(__cuda_arch_bin "35;37;50;52;53;60;61;62;70;72;75;80")
   elseif(NOT ${CUDA_VERSION_STRING} VERSION_LESS "10.0")
-    set(__cuda_arch_bin "3.0 3.2 3.5 3.7 5.0 5.2 5.3 6.0 6.1 6.2 7.0 7.2 7.5")
+    set(__cuda_arch_bin "30;32;35;37;50;52;53;60;61;62;70;72;75")
   elseif(NOT ${CUDA_VERSION_STRING} VERSION_LESS "9.0")
-    set(__cuda_arch_bin "3.0 3.2 3.5 3.7 5.0 5.2 5.3 6.0 6.1 6.2 7.0 7.2")
+    set(__cuda_arch_bin "30;32;35;37;50;52;53;60;61;62;70;72")
   endif()
```

Comment:
You could parse `nvcc --list-gpu-code` so we don't need to maintain this list (and maybe add `--Wno-deprecated-gpu-targets` or you find a way to detect deprecated architectures without parsing the `nvcc --help` ;-) ).